game.js

```diff
@@ -51,7 +59,10 @@ document.addEventListener('click', event => {
                 document.querySelector('.game-over').classList.add('visible')
                 document.querySelector('.game-over-text').textContent = xWins
                     ? 'X wins!'
-                    : 'O wins!'
+                    : 'O wins!';
+                game.gamesCounter++;
```

Comment:
same comment as above

Reply:
I moved it to the restart-after-gameover event because a new game count starts after a 'Game-over'.